README.md

```diff
@@ -10,37 +15,35 @@ pip install ismatec
 Usage
 =====
 
-### Command Line
+## Command Line
 
 ```
-$ ismatec <serial-to-ethernet-ip> --port <port> --channel 1
 $ ismatec /dev/ttyUSB0 --channel 2
+$ ismatec tcp://192.168.1.100:23 --channel 1
```

Comment:
Played with the API. I like leading with the serial example (will be the more common use case outside of us) and prefixing the IPs with a `tcp://` protocol to be more explicit about serial vs. ethernet.